azurelinuxagent/ga/exthandlers.py

```diff
@@ -1212,6 +1224,7 @@ def launch_command(self, cmd, timeout=300, extension_error_code=ExtensionErrorCo
                 # Always add Extension Path and version to the current launch_command (Ask from publishers)
                 env.update({ExtCommandEnvVariable.ExtensionPath: base_dir,
                             ExtCommandEnvVariable.ExtensionVersion: self.ext_handler.properties.version,
```

Comment:
Does this need to be cast to `str` too?

Reply:
I tried to ensure that this is always a string but was unable to do so. I'll add a str() cast just in case. Thanks for the catch!